### PR TITLE
add more owners for extensions

### PR DESCRIPTION
**Pull request by erwinmombay**
currently owners are only amp core team members as they are the only collaborators

### CODE REVIEW

File: extensions/amp-3q-player/OWNERS.yaml [hunk 1]
```diff
@@ -0,0 +1,2 @@
+- cvializ
```
Comment (aghassemi) [inline extensions/amp-3q-player/OWNERS.yaml]: let's put just alan for all video players

File: extensions/amp-delight-player/OWNERS.yaml [hunk 2]
```diff
@@ -0,0 +1 @@
+- aghassemi
```
Comment (aghassemi) [inline extensions/amp-delight-player/OWNERS.yaml]: alanorozco

File: extensions/amp-fit-text/OWNERS.yaml [hunk 3]
```diff
@@ -0,0 +1 @@
+- dvoytenko
```
Comment (aghassemi) [inline extensions/amp-fit-text/OWNERS.yaml]: aghassemi

File: extensions/amp-ima-video/OWNERS.yaml [hunk 4]
```diff
@@ -0,0 +1 @@
+- aghassemi
```
Comment (aghassemi) [inline extensions/amp-ima-video/OWNERS.yaml]: alanorozco
torch2424

File: extensions/amp-image-slider/OWNERS.yaml [hunk 5]
```diff
@@ -0,0 +1 @@
+- cathyxz
```
Comment (aghassemi) [inline extensions/amp-image-slider/OWNERS.yaml]: add
sparhami

File: extensions/amp-slides/OWNERS.yaml [hunk 6]
```diff
@@ -0,0 +1 @@
+- alanorozco
```
Comment (aghassemi) [inline extensions/amp-slides/OWNERS.yaml]: haha `amp-slides` :) 
change to sparhami

File: extensions/amp-subscriptions-google/OWNERS.yaml [hunk 7]
```diff
@@ -0,0 +1,2 @@
+- prateekbh
```
Comment (aghassemi) [inline extensions/amp-subscriptions-google/OWNERS.yaml]: jpettitt  instead of prateekbh?

File: extensions/amp-wistia-player/OWNERS.yaml [hunk 8]
```diff
@@ -0,0 +1 @@
+- aghassemi
```
Comment (aghassemi) [inline extensions/amp-wistia-player/OWNERS.yaml]: alanorozco

File: extensions/amp-3q-player/OWNERS.yaml (hunk 1 above)
Comment (erwinmombay) [reply extensions/amp-3q-player/OWNERS.yaml]: is there anyone else we can put in? would like an alternative reviewer if possible

File: extensions/amp-subscriptions-google/OWNERS.yaml (hunk 7 above)
Comment (choumx) [reply extensions/amp-subscriptions-google/OWNERS.yaml]: Use both. Also add `dvoytenko`.

File: extensions/OWNERS.yaml [hunk 9]
```diff
@@ -0,0 +1 @@
+- aghassemi
```
Comment (choumx) [inline extensions/OWNERS.yaml]: Do we need an extensions/ top-level?

File: extensions/amp-wistia-player/OWNERS.yaml (hunk 8 above)
Comment (erwinmombay) [reply extensions/amp-wistia-player/OWNERS.yaml]: done

File: extensions/amp-image-slider/OWNERS.yaml (hunk 5 above)
Comment (erwinmombay) [reply extensions/amp-image-slider/OWNERS.yaml]: done.

File: extensions/amp-ima-video/OWNERS.yaml (hunk 4 above)
Comment (erwinmombay) [reply extensions/amp-ima-video/OWNERS.yaml]: done.

File: extensions/amp-fit-text/OWNERS.yaml (hunk 3 above)
Comment (erwinmombay) [reply extensions/amp-fit-text/OWNERS.yaml]: done

File: extensions/amp-delight-player/OWNERS.yaml (hunk 2 above)
Comment (erwinmombay) [reply extensions/amp-delight-player/OWNERS.yaml]: done

File: extensions/amp-3q-player/OWNERS.yaml (hunk 1 above)
Comment (erwinmombay) [reply extensions/amp-3q-player/OWNERS.yaml]: done.

File: extensions/OWNERS.yaml (hunk 9 above)
Comment (erwinmombay) [reply extensions/OWNERS.yaml]: this is just a fallback so that reviews don't go to @cramforce

**Pull request merged**